upper footer and inicial home page

YasminDayrell opened the pull request.

Commit:
upper footer and inicial home page

jvopBR commented:
Alem das mudanças simples que o copilot sugeriu, eu aconselho mudança dos nomes de arquivo. Footer por si só é a parte de baixo do site, onde normalmente fica termos de uso, privacidade, nome da empresa, direito autorais e etc. E ele ja dá uma ideia de ser embaixo.

Ja a parte de cima, onde normalmente fica botões de navegação e etc é chamado de Header.

Tirando isso o código está simples e funcional, o que é muito bom.

Resumindo:

Upper_footer -> Header
LowerFooter -> Footer

jvopBR commented on nihon/src/app/layout.tsx:
Caso haja mudança no nomes dos arquivos. Não esquecer de chamar novamente como nome corrigido

jvopBR commented on nihon/src/components/layout/UpperFooter.tsx:
Nome do arquivo, o codigo em si está correto e funcionando

jvopBR reviewed (changes requested):
Acho que essas mudanças são importantes afim de facilitar a identificação dos arquivos e escrita deles durante codigos futuros

jvopBR committed:
New Foote

Commit:
supliers home bar

Commit:
home page products

jvopBR committed:
Card Animation, Added framer motion

Commit:
page buttons

Commit:
small changes at footer and small add ins to home page

jvopBR committed:
Footer update/Home page categories

GFlyan committed:
Added Content

I've added the rounded Nihon Logotype as it's on Figma and added some sections for put text and images.

GFlyan committed:
Add Content

I've added some hooks in the header function causing hidden effect and some other contents about the stylish on the layout.tsx.

Commit:
add banner arte

jvopBR committed:
New suppliers bar animation

jvopBR committed:
Merge branch 'upper_footer_dev' of https://github.com/mdsreq-fga-unb/REQ-2025.2-T02-Nihon into upper_footer_dev

jvopBR committed:
New suppliers animation

GFlyan committed:
Add Content

I've added the store's photo and the localization in the Google Maps and added content about the enterprise.

Marjoriemitzi committed:
Create .github

Marjoriemitzi committed:
Delete .github

Marjoriemitzi committed:
Create .github

Marjoriemitzi committed:
Delete .github

Marjoriemitzi committed:
Create pull_request_template.md

GFlyan committed:
Added Content

I've created some components to reduce the code in the About function, and finished all the components that will be used in the about page.

brun0duarte committed:
feat(about): improve Location and Values sections layout with hover effects and color fixes

- Add hover effects to LocationSection contact cards and interactive elements
- Improve Location Section layout and content structure with horizontal orientation
- Improve OurValues Section layout and content structure with better spacing
- Fix color inconsistencies across components and theme

Commit:
fix(redirect): set overflow-y-hidden

Commit:
feat: add scroll to top button

Commit:
Home page improviments

Commit:
adding categories imagens and fixing some errors

Commit:
fixing supliers banner error

Marjoriemitzi committed:
Merge pull request #12 from mdsreq-fga-unb/feature/-organization

Feature/ organization

Commit:
feat(layout): implement responsive header, footer, and navigation

This commit refactors the Header and Footer components. It implements a responsive design, including a hamburger menu for mobile, and adds a scroll-to-top button for better user experience.

brun0duarte committed:
Merge branch 'feature/about' into develop

brun0duarte committed:
feat: add links to whatsapp, instagram and maps
